Makefile

```diff
@@ -293,22 +276,29 @@ $(BUILT)/standalone/content/l10n/%/loop.properties: locale/%/standalone.properti
 
 # add-on
 add_on_jsx_files=$(wildcard add-on/panels/js/*.jsx)
+add_on_js_files=$(wildcard add-on/panels/js/*.js)
+add_on_test_files=$(wildcard add-on/panels/test/*.js)
+add_on_module_jsm_files=$(wildcard add-on/chrome/modules/*.jsm)
+add_on_module_js_files=$(wildcard add-on/chrome/modules/*.js)
 built_add_on_js_files=$(patsubst add-on/panels/js/%.jsx, \
 	 $(BUILT)/add-on/chrome/content/panels/js/%.js, \
-	 $(add_on_jsx_files))
-
-$(BUILT)/add-on/chrome/content/panels/js/%.js: add-on/panels/js/%.jsx
-	@mkdir -p $(@D)
-	$(BABEL) $< --out-file $@
+	 $(add_on_jsx_files)) $(patsubst add-on/panels/js/%.js, \
+ 	 $(BUILT)/add-on/chrome/content/panels/js/%.js, \
+ 	 $(add_on_js_files)) $(patsubst add-on/panels/test/%.js, \
+ 	 $(BUILT)/add-on/chrome/content/panels/test/%.js, \
+ 	 $(add_on_test_files)) $(patsubst add-on/chrome/modules/%.jsm, \
+ 	 $(BUILT)/add-on/chrome/content/modules/%.jsm, \
+ 	 $(add_on_module_jsm_files)) $(patsubst add-on/chrome/modules/%.js, \
+ 	 $(BUILT)/add-on/chrome/content/modules/%.js, \
+ 	 $(add_on_module_js_files))
```

Comment:
Looking more closely at each of the `patsubst`, it seems like you just want to expand `panels/` to `chrome/panels/` then `chrome/` to `chrome/content/` then prefix with `built/`?
```makefile
add_on_js_files=$(wildcard add-on/panels/js/*.js)
add_on_jsx_files=$(wildcard add-on/panels/js/*.jsx)
add_on_module_js_files=$(wildcard add-on/chrome/modules/*.js)
add_on_module_jsm_files=$(wildcard add-on/chrome/modules/*.jsm)
add_on_test_files=$(wildcard add-on/panels/test/*.js)
add_on_files=$(add_on_js_files) $(add_on_jsx_files:.jsx=.js) $(add_on_module_js_files) $(add_on_module_jsm_files) $(add_on_test_files)
built_add_on_js_files=$(addprefix $(BUILT)/, $(subst chrome/,chrome/content/, $(subst panels/,chrome/panels/, $(add_on_files))))
```